samples/csharp_dotnetcore/48.customQABot-all-features/Dialogs/RootDialog.cs

```diff
@@ -0,0 +1,121 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using AdaptiveExpressions.Properties;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Builder.AI.QnA.Dialogs;
+using Microsoft.Bot.Builder.AI.QnA.Models;
+using Microsoft.Bot.Builder.Dialogs;
+using Microsoft.Extensions.Configuration;
+
+namespace Microsoft.BotBuilderSamples.Dialogs
```

Comment:
Hi @gabog ,
To simplify Bot.Builder.Dialog used in CustomQABot.cs and to maintain parity with CoreBot, we have changed namespace and folder name to "Dialogs".
Earlier it was "Dialog"